distributions/otelcol-paas/README.md

```diff
@@ -0,0 +1,73 @@
+# OpenTelemetry Collector PaaS Distro
+
+This distribution is made specifically to be used in a PaaS environment. It contains a subset of components from [OpenTelemetry Collector Core](https://github.com/open-telemetry/opentelemetry-collector) and [OpenTelemetry Collector Contrib](https://github.com/open-telemetry/opentelemetry-collector-contrib).
+
+## Components
+
+The full list of components is available in the [manifest](manifest.yaml).
+
+### Rules for Component Inclusion
+
+ - Only includes components from Contrib and Core.
+ - Only components that are Alpha or higher.
+ - All receivers must facilitate the collection of data that is generated by the application running in the PaaS environment or enriching the data by introspecting the PaaS platform.
+ - All components must be vendor-neutral.
+ - Only exporters that use OTLP are allowed.
+   - To facilitate troubleshooting, the nop and debug exporters are present as exceptions.
+
+## Heroku buildpack
+
+This distribution works as a Heroku buildpack.
+
+### Installation
+
+[Install the Heroku CLI, login, and create an
+app](https://devcenter.heroku.com/articles/heroku-cli). Add and configure the
+buildpack:
+
+```
+# cd into the Heroku project directory
+# WARNING: running `heroku` command outside of project directories
+#          will result in unexpected behavior
+cd <HEROKU_APP_DIRECTORY>
+
+# Configure Heroku App to expose Dyno metadata
+# This metadata is required by the OpenTelemetry Collector to
+# set global dimensions such as `app_name`, `app_id` and `dyno_id`.
+# See [here](https://devcenter.heroku.com/articles/dyno-metadata) for more information.
+heroku labs:enable runtime-dyno-metadata
+
+# Add buildpack for OpenTelemetry Collector
+# Note both lines are required together
+# TODO check URI works
+heroku buildpacks:add https://github.com/open-telemetry/opentelemetry-collector-releases.git/distributions/otelcol-paas#\
```

Comment:
I didn't test this step myself yet, I will try this and get the PR out of draft when I know this works well.